bulbs/special_coverage/search.py

```diff
@@ -1,27 +1,110 @@
 """Custom search function to assist in customizing our special coverage searches."""
 import six
 
+from elasticsearch_dsl import filter as es_filter
 
-def second_slot_query_generator(query1, query2):
-    """Returns the result of a different query at the 1st index of iteration.
-
-    :param query1: Primary search that will be the default result set.
-    :type  query1: Any iterable object; intended for djes.search.LazySearch and django.Querysets.
-    :param query2: Secondard search that will return at the 1st index of iteration.
-    :type  query2: Any iterable object; intended for djes.search.LazySearch and django.Querysets.
-    """
-    index = 0
-    while True:
-        result = None
-        if index == 1:
-            try:
-                result = six.next(query2)
-            except IndexError:
-                pass
-        if result is None:
-            try:
-                result = six.next(query1)
-            except IndexError:
-                break
-        yield result
-        index += 1
+from bulbs.content.custom_search import get_condition_filter
+from bulbs.content.models import Content
+
+
+class SearchParty(object):
+    """Wrapper that searches for content from a group of SpecialCoverage objects."""
+
+    def __init__(self, special_coverages, *args, **kwargs):
```

Comment:
Might as well drop *args + **kwargs since not using them, else they could mask unused params passed by caller